src/main/handlers/IpcHandler.ts

```
@@ -1,11 +1,523 @@
+      })
+.on('error', (err) => {
+        if (silentWavPath) {
+          fs.promises.unlink(silentWavPath).catch(() => {
+            /* ignore cleanup errors */
+          })
+        }
+        reject(new Error(`FFmpeg error (${encoder.name}): ${err.message}`))
+      })
+      .output(videoPath)
+      .run()
+  })
```

Biome (2.4.13)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)